checks/zookeeper.go

```diff
@@ -55,3 +65,65 @@ func (chk ZooKeeperRUOK) Status() (int, string, error) {
 	}
 	return 1, "Failed: " + failed, nil
 }
+
+type ZooKeeperServerStats struct {
+	timeout time.Duration
+	servers []string
+    minLatency int64
+    maxLatency int64
+    avgLatency int64
+}
+
+
+func (chk ZooKeeperServerStats) New(params []string) (chkutil.Check, error) {
+	if len(params) < 5 {
+		return chk, errutil.ParameterLengthError{2, params}
```

Comment:
Is bit rewritten your code,  4 positional parameters, + server list (as in RUOK module)